### PR TITLE
update api docs

### PR DESCRIPTION
**Pull request by runzexia**
Signed-off-by: runzexia <runzexia@yunify.com>

### CODE REVIEW

File: pkg/apis/devops/v1alpha2/register.go [hunk 1]
```diff
@@ -50,34 +50,34 @@ func addWebService(c *restful.Container) error {
...
-		Doc("Get devops project").
+		Doc("Get a DevOps Project").
 		Metadata(restfulspec.KeyOpenAPITags, tags).
-		Param(webservice.PathParameter("devops", "devops project's Id")).
+		Param(webservice.PathParameter("devops", "DevOps Project's Id, e.g. project-RRRRAzLBlLEm")).
 		Returns(http.StatusOK, RespOK, devops.DevOpsProject{}).
 		Writes(devops.DevOpsProject{}))
 
 	webservice.Route(webservice.PATCH("/devops/{devops}").
 		To(devopsapi.UpdateProjectHandler).
```
Comment (rayzhou2017) [inline pkg/apis/devops/v1alpha2/register.go]: still keep /devops/？

Comment (runzexia) [reply pkg/apis/devops/v1alpha2/register.go]: This is part of the modification

File: pkg/apis/devops/v1alpha2/register.go [hunk 2]
```diff
@@ -88,131 +88,131 @@ func addWebService(c *restful.Container) error {
 		Returns(http.StatusOK, RespOK, []devops.DevOpsProjectMembership{}).
 		Writes([]devops.DevOpsProjectMembership{}))
 
-	webservice.Route(webservice.GET("/devops/{devops}/members/{members}").
+	webservice.Route(webservice.GET("/devops/{devops}/members/{member}").
 		To(devopsapi.GetDevOpsProjectMemberHandler).
-		Doc("Get devops project member").
+		Doc("Get a Member of the DevOps Project").
```
Comment (rayzhou2017) [inline pkg/apis/devops/v1alpha2/register.go]: Member --> member

File: pkg/apis/devops/v1alpha2/register.go [hunk 3]
```diff
@@ -88,131 +88,131 @@ func addWebService(c *restful.Container) error {
...
-	webservice.Route(webservice.PUT("/devops/{devops}/credentials/{credentials}").
+	webservice.Route(webservice.PUT("/devops/{devops}/credentials/{credential}").
 		To(devopsapi.UpdateDevOpsProjectCredentialHandler).
-		Doc("Update project credential pipeline").
+		Doc("Update a Credential under the DevOps Project").
 		Metadata(restfulspec.KeyOpenAPITags, tags).
-		Param(webservice.PathParameter("devops", "devops project's Id")).
-		Param(webservice.PathParameter("credentials", "credential's Id")).
+		Param(webservice.PathParameter("devops", "DevOps Project's Id, e.g. project-RRRRAzLBlLEm")).
+		Param(webservice.PathParameter("credentials", "Credential's Id, e.g. dockerhub-id")).
```
Comment (rayzhou2017) [inline pkg/apis/devops/v1alpha2/register.go]: credentials --> credential

**Pull request merged**